src/js/common/utils/helpers.js

```diff
@@ -101,14 +101,33 @@ export function dateToMoment(dateField) {
   });
 }
 
+export function dropTimeAndTimezone(date) {
+  let newDate = moment(date).format();
+  const indexOfTime = newDate.indexOf('T');
```

Comment:
This is because you're passing that date into `moment()` straight away on line 105.